webpack/assets/javascripts/react_app/components/common/ActionButtons/ActionButtons.js

```diff
@@ -1,34 +1,81 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
-import { SplitButton, MenuItem, Button } from 'patternfly-react';
+import {
+  Button,
+  Dropdown,
+  DropdownItem,
+  DropdownList,
+  MenuToggle,
+  MenuToggleAction,
+} from '@patternfly/react-core';
+import './actionButtons.scss';
 
 /**
  * Generate a button or a dropdown of buttons
  * @param  {String} title The title of the button for the title and text inside the button
  * @param  {Object} action action to preform when the button is click can be href with data-method or Onclick
- * @return {Function} button component or splitbutton component
+ * @return {Function} button component or dropdown with menu toggle action component
  */
 export const ActionButtons = ({ buttons }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const onToggleClick = () => setIsOpen(!isOpen);
+
   if (!buttons.length) return null;
   if (buttons.length === 1)
     return (
-      <Button bsSize="small" {...buttons[0].action}>
+      <Button
+        ouiaId="action-button"
+        size="sm"
+        variant="primary"
+        {...buttons[0].action}
```

Comment:
Some components use disable action prop, which is not valid for PF button, something like this, or remapping should work.

```suggestion
        isDisabled={buttons[0].action?.disabled}
        {...buttons[0].action}
```

Reply:
I added the change. Thanks